site/content/vendir/docs/develop/vendir-spec.md

```diff
@@ -14,6 +14,11 @@ directories:
 - # path is relative to `vendir` CLI working directory
   path: config/_ytt_lib
 
+  # set the permissions for this directory (optional; v0.32.3+)
```

Comment:
version of vendir should be v0.33.0+ since we wouldnt want to add a feature in a patch release. /cc @joaopapereira